lib/kino_slack.ex

```diff
@@ -1,18 +1,84 @@
 defmodule KinoSlack do
```

Comment:
Let's use a nested module, like `KinoSlack.MessageCell`, so that we can have more if we want. Also `@moduledoc false`, since there is no public API :)